app/views/uploads/_value.html.erb

```diff
@@ -0,0 +1,61 @@
+<% if value[:type] == 'ANYNonNull' %>
+<!-- any value -->
```

Comment:
Is this if block supposed to be empty? If so you can just flip the condition and there's no need for an else block.